demos/serverless/template.yaml

```diff
@@ -32,6 +32,20 @@ Parameters:
     Description: Prefix of S3 bucket to write capture artifacts.  The bucket name will be suffixed with the region.
     Default: ""
     Type: String
+  MediaPipelinesControlRegion:
+    Description: The AWS SDK Chime Media Pipelines control REGION
+    Default: "us-east-1"
+    Type: String
+  ChimeSDKMediaPipelinesEndpoint:
+    Description: The AWS Chime SDK Media Pipelines endpoint
+    Default: "https://media-pipelines-chime.us-east-1.amazonaws.com"
+    Type: String
+  UseChimeSDKMediaPipelines:
+    Description: Use boolean to switch between Chime and ChimeSDKMediaPipelines Client
+    Default: true
```

Comment:
[Clarification] Do these `true`|`false` need to be in string? Like "true".

Reply:
no: https://github.com/aws/amazon-chime-sdk-js/blob/main/demos/serverless/template.yaml#L18